api/openapi.gen.json

```diff
@@ -33,6 +33,32 @@
       }
     },
     "schemas": {
+      "v1.AWSReservationResponse": {
+        "properties": {
+          "amount": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "aws_reservation_id": {
+            "type": "string"
+          },
+          "image_id": {
+            "type": "string"
+          },
+          "instance_type": {
+            "type": "string"
+          },
+          "pubkey_id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "source_id": {
+            "format": "int64",
+            "type": "integer"
+          }
+        },
+        "type": "object"
+      },
```

Comment:
I am missing one very important field - reservation id. That is really needed for doing followup updates on the status.

Reply:
the struct includes ReservationResponsePayload which has the id. 
I see that none of the ReservationResponsePayload fields are shown..

Reply:
added :)